gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-9.0.0-rc-3-bin.zip
```

Comment:
Gradle 9.0 released so we could update this to not use an rc. I'm also okay with keeping it at 8.7 as long as it's compatible with 9+

Reply:
I updated this to `9.0.0`.